### PR TITLE
Upgrade to 14.0.0

### PR DESCRIPTION
**Pull request by Mudit-G**

### CODE REVIEW

File: workers/csharp_physics/src/Main.cs [hunk 1]
```diff
@@ -89,15 +89,16 @@ private static int Main(string[] args)
...
+                    connection.SendComponentUpdate(Position.Metaclass, ourEntity, positionUpdate);
 
                     // Sleep for some time.
                     System.Threading.Thread.Sleep(1000);
 
                     // Send a command every 2 ticks.
                     if (tickCount % 2 == 0)
                     {
-                        connection.SendCommandRequest(ourEntity, new ClientData.Commands.TestCommand.Request(10, 1.5f), null);
+                        connection.SendCommandRequest(ClientData.Commands.TestCommand.Metaclass,
```
Comment (dgavedissian) [inline workers/csharp_physics/src/Main.cs:100]: Hmm. I think the upgrade guide I wrote for changing to the new C# syntax might not be 100% correct when changing the request parameter (I wrote something like `new TestCommand.Request(new Sample.AddCommandRequest(args))`, when I should've written `new TestCommand.Request(args)` cause the old wrapper type has a constructor that forwards the constructor arguments to the concrete type). Maybe it's fine and users will get it, and I can just fix it when back on the 22nd.

File: workers/c_client_vtable/client.c [hunk 2]
```diff
@@ -173,8 +173,7 @@ int main(int argc, char** argv) {
   command_request.user_handle = CreateCommandObject(1, &take_control_request);
```
Comment (dgavedissian) [inline workers/c_client_vtable/client.c:174]: Don't forget to set `command_request.command_index = 1;` here.

File: workers/c_client_direct/client.c [hunk 3]
```diff
@@ -139,8 +139,7 @@ int main(int argc, char** argv) {
   command_request.schema_type = Schema_CreateCommandRequest(LOGIN_COMPONENT_ID, 1);
   Worker_CommandParameters command_parameters;
```
Comment (dgavedissian) [inline workers/c_client_direct/client.c:141]: `command_request.command_index = 1;` here too.

**Review by dgavedissian — APPROVED** [review]

File: workers/c_client_vtable/client.c (hunk 2 above)
Comment (Mudit-G) [reply workers/c_client_vtable/client.c:174]: Good spot, fixed.

File: workers/c_client_direct/client.c (hunk 3 above)
Comment (Mudit-G) [reply workers/c_client_direct/client.c:141]: Fixed.

File: workers/csharp_physics/src/Main.cs (hunk 1 above)
Comment (Mudit-G) [reply workers/csharp_physics/src/Main.cs:100]: You wrote
```cs
// before
var request = new MyRequest(123, 89.0);
connection.SendCommandRequest<MyComponent.Commands.TestCommand>(entityId, new MyComponent.Commands.TestCommand.Request(request), ...);
// after
var request = new MyRequest(123, 89.0);
connection.SendCommandRequest(MyComponent.Commands.TestCommand.Metaclass, entityId, request, ...);
```
which I think is correct?

**Pull request merged**